css-typed-om/Overview.bs

```diff
@@ -933,7 +933,9 @@ Shorthand properties and values are not supported.
 
 Issue(159): Spec up ColorValue
 
-Issue(146): Do we need to say that LengthValues must be positive e.g. for border-widths here?
+{{LengthValue}} objects are not range-restricted. Any valid number can be represented by a {{LengthValue}},
+and that value will not be clamped, rounded, or rejected when set on a <a>specified StylePropertyMap</a> or
+<a>inline StylePropertyMap</a>. Instead, clamping and/or rounding will occur during computation of style.
```

Comment:
remove the 'Instead', part - replace with
'Properties which can't accept the full range of length values (e.g. 'border-width') perform clamping and/or rounding during style computation.'.